ios/RNTPTDocumentView.m

```diff
@@ -1842,4 +1851,25 @@ - (void)toolManagerDidRemoveAnnotationWithNotification:(NSNotification *)notific
     }
 }
 
+- (void)toolManagerDidModifyFormFieldDataWithNotification:(NSNotification *)notification
+{
+    if (notification.object != self.documentViewController.toolManager) {
+        return;
+    }
+
+    PTAnnot *annot = notification.userInfo[PTToolManagerAnnotationUserInfoKey];
+
+    PTWidget *widget = [[PTWidget alloc] initWithAnn:annot];
+    PTField *field = [widget GetField];
+    NSString *fieldName = [field IsValid] ? [field GetName] : @"";
+    NSString *fieldValue = [field IsValid] ? [field GetValueAsString] : @"";
```

Comment:
Maybe wrap this in a `pdfViewCtrl.DocLockReadWithBlock:error:`?
Could also be good to double check that the annot type is correct for a widget.